src/actions/expressions.js

```diff
@@ -38,7 +38,8 @@ export function addExpression(input: string) {
 
     const selectedFrame = getSelectedFrame(getState());
     const selectedFrameId = selectedFrame ? selectedFrame.id : null;
-    dispatch(evaluateExpression({ input }, selectedFrameId));
+    const newExpression = getExpression(getState(), input);
+    dispatch(evaluateExpression(newExpression, selectedFrameId));
```

Comment:
when i made the flow type stricter I realized i wasn't passing a real expression into evaluate :)

Reply:
👍